dataprovider/src/main/java/cm/aptoide/pt/dataprovider/model/v7/timeline/AppUpdate.java

```diff
@@ -1,74 +0,0 @@
-/*
- * Copyright (c) 2016.
- * Modified by Marcelo Benites on 01/07/2016.
- */
-
-package cm.aptoide.pt.dataprovider.model.v7.timeline;
-
-import cm.aptoide.pt.dataprovider.model.v7.listapp.App;
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-/**
- * Created by marcelobenites on 7/1/16.
- */
-public class AppUpdate extends App implements TimelineCard {
-
-  private final String cardId;
-  private final Ab ab;
```

Comment:
Ab doesn't have any usages, Inside the Ab class there is another Conversion class that doesn't have any usages

Reply:
deleted